src/System.Net.Mime/src/System/Net/Mime/MimeBasePart.cs

```diff
@@ -0,0 +1,289 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Specialized;
+using System.Text;
+using System.Net.Mail;
+
+namespace System.Net.Mime
+{
+    internal class MimeBasePart
+    {
+        internal const string DefaultCharSet = "utf-8";//"iso-8859-1";
```

Comment:
No idea.  I'm going to avoid removing such things until all consumers of this code are actually ported and we know what we're dealing with.  Most likely the comment could be removed.